rotors_gazebo_plugins/include/rotors_gazebo_plugins/gazebo_noisydepth_plugin.h

```diff
@@ -0,0 +1,160 @@
+/*
```

Comment:
I think I would slightly prefer gazebo_noisy_depth_plugin, but you can also keep this ;)